try {
	// Code that can potentially throw an exception
} catch (Exceptioname1 e) {
	// Execution of operations if an exception 1 is caught
} catch (Exceptioname2 e) {
	// Execution of operations if an exception 2 is caught
}
